app/assets/stylesheets/omni_nav/omni_nav_v2/omni-nav.scss

```diff
@@ -428,15 +428,15 @@ html.omni-nav-v2 {
             }
           }
 
+          .nav-icon-text {
```

Comment:
This fixes the issue from this story: https://trello.com/c/dlxo1Bnp

The bug was preventing me from testing the "close" button on the emergency alert on mobile, so I fixed it.